arch/x86_64/kernel/kernel.cpp

```diff
@@ -39,85 +39,6 @@ void write_ff_info() {
 }
 }  // namespace firefly::kernel::main
 
```

Comment:
why'd u remove printf

Reply:
that was a printf I put in for testing and never removed because I simply forgot.
The real printf is in `stl/cstdlib/stdio.cpp` and is platform independent

Reply:
okay.